scripts/ci/install-minikube.sh

```diff
@@ -101,10 +101,10 @@ start() {
 
         local args="--kubernetes-version $K8S_VERSION --memory 4096"
         if [ "$MINIKUBE_DRIVER" == "none" ]; then
-                args="$args --vm-driver=none --bootstrapper=localkube"
+                args="$args --vm-driver=none"
```

Comment:
This was for the vagrant box. Did you test it ?

Reply:
yes